tests/offscreencanvas_tests.ts

```diff
@@ -0,0 +1,60 @@
+// [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2021.
+// MIT License
+//
+// OffscreenCanvas Tests
+
+import { CanvasContext } from '../src/canvascontext';
+import { Formatter } from '../src/formatter';
+import { Stave } from '../src/stave';
+import { BarlineType } from '../src/stavebarline';
+import { StaveNote } from '../src/stavenote';
+
+const OffscreenCanvasTests = {
+  Start(): void {
+    // At the time of writing, OffscreenCanvas is still an experimental technology.
+    if (window.OffscreenCanvas === undefined) {
```

Comment:
It's probably fine since we mostly run tests in the browser, but maybe someday we need a util method for getting the global, to avoid 'window' is not defined scenarios.